### PR TITLE
Tweak: Adjust all external system links to include `noopener` attribute [ED-6580]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: modules/admin-top-bar/assets/js/components/connection-button/connection-button.js [hunk 1]
```diff
@@ -16,17 +16,19 @@ export default function ConnectionButton() {
 	let tooltipText = __( 'Connect your account to get access to Elementor\'s Template Library & more.', 'elementor' ),
 		connectUrl = elementorAdminTopBarConfig.connect_url,
 		buttonText = __( 'Connect Account', 'elementor' ),
-		targetUrl = '_self';
+		targetUrl = '_self',
+		rel = '';
```
Comment (Nevoss) [inline modules/admin-top-bar/assets/js/components/connection-button/connection-button.js:20]:
```suggestion
		rel = null;
```

Comment (Nevoss) [reply modules/admin-top-bar/assets/js/components/connection-button/connection-button.js:20]: Did you check that it is working?

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: modules/admin-top-bar/assets/js/components/connection-button/connection-button.js (hunk 1 above)
Comment (StyleShit) [reply modules/admin-top-bar/assets/js/components/connection-button/connection-button.js]: Done

**Review by Nevoss — APPROVED** [review]

**Comment by matipojo** [comment]
The modern browsers are already handling it:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#:~:text=target%3D%22_blank%22%20implies%20rel%3D%22noopener%22%20behavior

**Pull request closed without merging**